include/hip/nvcc_detail/hip_runtime_api.h

```diff
@@ -1679,6 +1686,17 @@ inline static hipError_t hipGetChannelDesc(hipChannelFormatDesc* desc, hipArray_
     return hipCUDAErrorTohipError(cudaGetChannelDesc(desc,array));
 }
 
+inline static hipError_t hipLaunchCooperativeKernel(const void* f, dim3 gridDim, dim3 blockDimX,
```

Comment:
Really minor comment, but should the blockDimX variable instead be named blockDim? The X seems to be a copy/paste from another function.

Reply:
Agree, rename it.